qtaf-core/src/main/resources/qtaf.json

```diff
@@ -32,9 +32,9 @@
     "projectKey": null,
     "authentication": {
       "xray": {
-        "clientId": null,
-        "clientSecret": null,
-        "bearerToken": null
+        "clientId": "your-client-id",
+        "clientSecret": "your-client-secret",
+        "bearerToken": "your-bearer-token"
```

Comment:
In my opinion, `null` is the better default since the configuration map explicitly checks for `null` values and prints corresponding info/error messages.

Reply:
Set values to null